### DISCUSSION TITLE
Community ideas contest for democratizing machine learning

### DISCUSSION THREAD

**Discussion by C0staTin** [mindsdb/mindsdb#2277] [Announcements]
Albert Einstein, a master at great ideas, once said "If at first the idea is not absurd, then there is no hope for it". 
Have an idea for MindsDB integrations or machine learning models? 
Check-out the new [MindsDB ideation💡 challenge](https://github.com/mindsdb/mindsdb/blob/staging/integrations_contest.md) (and the cash💵 prizes)!

![1,19_1Ideas Contest](https://user-images.githubusercontent.com/12409467/171374970-5aa76e6c-d1c6-4896-92e2-20b33ac8875b.png)

**Comment by reetukumar** [comment]
I’m excited to uncover this page. I need to thank you for your time for this, particularly fantastic read!! I definitely really liked every part of it and I also have you saved to list to look at new information on your site.


Click to know more: [Machine Learning Classes in Pune](https://www.sevenmentor.com/machine-learning-course-in-pune.php)
